app/views/games/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <h1><%= @game.name %></h1>
 <p>Players: <%= @game.min_players %> to <%= @game.max_players %></p>
 <p>Playing time: <%= @game.playing_time %></p>
-<p>Complexity: <%= @game.complexity %></p>
+<p>Complexity: <%= Game::COMPLEXITIES[@game.complexity] %></p>
```

Comment:
Would that work as written? I think it would have to be complexity_name(complexity), at which point it's not really any shorter to call than what we did?